pallets/registrar/src/benchmarks.rs

```diff
@@ -375,7 +447,7 @@ mod benchmarks {
         }
 
         // Second loop to fill RegisteredParaIds to its maximum
-        for k in 1000..(1000 + y) {
+        for k in 1000..(1000 + y - 1) {
```

Comment:
This `-1` is because if we will `RegisteredParaIds` to its maximum, the call below will fail because the list is full, so we need to leave space for 1 chain.

Reply:
Maybe it's good to add this as a comment in the code?